sql/healthpro/salivary_inclusion_zip_codes.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE `salivary_inclusion_zip_codes` (
```

Comment:
We have a migration for the data for this table (`sql/migrations/033-insert-salivary-inclusion-zip-codes.sql`), but nothing that would bootstrap it for a new development environment or unit tests (two cases where we usually don't run the migrations). Not sure if this has to be addressed as part of this PR, but it may call for us to handle that differently in the future if this is a critical data fixture.

Reply:
Decided to remove these migration files completely as we no longer needed them. Thanks!